README.md

```diff
@@ -27,19 +27,58 @@ If you have a capable (Nvidia, at least 8GB of VRAM) GPU, it is highly recommend
 
 ## Training a model
 
-Open a terminal, change to the directory of the DeepSpeech checkout and run
+The central (Python) script is `DeepSpeech.py` in the project's root directory:
```

Comment:
I think it would be useful to prepare the reader for what to expect from this call, giving them some context for why they are doing this command line call.